src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/organization_policy.py

```diff
@@ -116,33 +116,34 @@ def apply(self, organizations, parameter_store, config):  # pylint: disable=R091
                     )
                     if stored_policy not in _policies:
                         OrganizationPolicy.clean_and_remove_policy_attachment(
-                            organization_mapping, path, organizations, _type)
+                            organization_mapping, path, policy_filename, organizations, _type)
             except ParameterNotFoundError:
                 LOGGER.debug(
                     'Parameter %s was not found in Parameter Store, continuing.', policy)
                 pass
 
             for _policy in _policies:
```

Comment:
Could you change the name of the variable `_policy` to `policy_path`. And change `_policies` to `policy_paths`.
I don't know why these are using underscores in front, as they are not class/instance variables that need protection, would be good to revert that. I hope you don't mind making these small improvements to the file on the side 🤞. If you would rather not, I am happy to push another PR for that later too.

Reply:
Made these changes.